docs/design/proposals/rbd-snap-clone.md

```diff
@@ -18,42 +18,42 @@
     - [RBD CLI commands to create a Volume from Volume](#rbd-cli-commands-to-create-a-volume-from-volume)
 
 This document outlines the command used to create RBD snapshot, delete RBD
-snapshot Restore RBD snapshot and Create New RBD image from RBD image.
+snapshot, Restore RBD snapshot and Create new RBD volume from existing RBD volume.
```

Comment:
rbd volume should be `rbd image`  as RBD represents it's as an image

Reply:
sure. thanks . done @Madhu-1